images/php-fpm/8.0.Dockerfile

```diff
@@ -65,23 +64,39 @@ RUN apk add --no-cache fcgi \
         # for yaml
         yaml-dev \
         # for imagemagick
+        imagemagick-dev \
+        && apk add --no-cache --virtual .phpize-deps $PHPIZE_DEPS \
+        && yes '' | pecl install -f apcu-5.1.21 \
+        && yes '' | pecl install -f imagick-3.7.0 \
+        && yes '' | pecl install -f redis-5.3.7 \
+        && yes '' | pecl install -f xdebug-3.1.3 \
+        && yes '' | pecl install -f yaml-2.2.2 \
+        && docker-php-ext-enable apcu imagick redis xdebug yaml \
+        && rm -rf /tmp/pear \
+        && apk del -r .phpize-deps \
+        && sed -i '1s/^/;Intentionally disabled. Enable via setting env variable XDEBUG_ENABLE to true\n;/' /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
+        && docker-php-ext-configure gd --with-webp --with-jpeg \
+        && docker-php-ext-install -j4 bcmath gd gettext mysqli pdo_mysql opcache pdo_pgsql pgsql shmop soap sockets xsl zip \
+        # PHP failing to build sockets extension https://github.com/docker-library/php/issues/1245#issuecomment-1020146157
+        && CFLAGS="$CFLAGS -D_GNU_SOURCE" docker-php-ext-install -j4 sockets \
```

Comment:
these are no longer needed - you must have an older branch!

Reply:
oh yeah I thought I removed that :facepalm: